.github/labeler.yml

```diff
@@ -136,6 +136,10 @@
   - plugins/postcss-logical/**
   - experimental/postcss-logical/**
 
+"plugins/postcss-logical-float-and-clear":
+  - plugins/postcss-logical/**
+  - experimental/postcss-logical/**
```

Comment:
```suggestion
"plugins/postcss-logical-float-and-clear":
  - plugins/postcss-logical-float-and-clear/**
  - experimental/postcss-logical-float-and-clear/**
```

Reply:
I also forgot about these :)
Can you also add this plugin to the issue templates?